portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -696,13 +696,40 @@ definition {
 	}
 
 	@description = "Verify that a Form can be deleted"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDeleted {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-130509 CanBeDeleted pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
```

Comment:
Hi @fmarcio, based on Diogo's announcement, could you please change it to adding a Text field instead of a Paragraph field? Thanks!

Reply:
@brittneyq fixed! :-)